src/ConfigEditor/DataSourceDescription.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { css } from '@emotion/css';
+import { useTheme2 } from '@grafana/ui';
+
+type Props = {
+  dataSourceName: string;
+  docsLink: string;
+  hasRequiredFields?: boolean;
+};
+
+export const DataSourceDescription: React.FC<Props> = ({ dataSourceName, docsLink, hasRequiredFields = true }) => {
```

Comment:
As Nathan pointed out here https://github.com/grafana/grafana/pull/67014#discussion_r1178546182 I think we're moving away from using FC

Reply:
It's the first time I heard about it. There's an article and PR referenced in Nathan's comment though.

Reply:
Updated in 56fc4cb8348fcc370b489ebd5b6f276248590281